package.json

```diff
@@ -56,7 +56,7 @@
     "d3-interpolate": "^3.0.1",
     "d3-scale": "^4.0.2",
     "d3-selection": "^3.0.0",
-    "d3-shape": "^3.1.0",
+    "d3-shape": "^2.1.0",
```

Comment:
Is there a reason we need to downgrade these packages?

Reply:
No. I do for my library which has some old conflicts, but I did *not* mean for that to be included in the pull request.The change should only have the one letter change from the first commit.